pkg/cmd/codespace/delete.go

```diff
@@ -143,6 +142,13 @@ func (a *App) Delete(ctx context.Context, opts deleteOptions) error {
 	if err := g.Wait(); err != nil {
 		return errors.New("some codespaces failed to delete")
 	}
+
+	noun := "Codespace"
+	if len(codespacesToDelete) > 1 {
+		noun = noun + "s"
+	}
+	a.logger.Println(noun + " deleted.")
```

Comment:
Just to confirm are we wanting this to stay in prod code or is this something that will printed to the CLI output that the developer sees?

Reply:
This is printed to the user. It uses to be there and was removed in a previous refactor.